lxd/storage_volumes_config.go

```diff
@@ -0,0 +1,125 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/lxc/lxd/shared"
+	"github.com/lxc/lxd/shared/api"
+)
+
+var storageVolumeConfigKeys = map[string]func(value string) error{
+	"block.mount_options":  shared.IsAny,
+	"block.filesystem":     shared.IsAny,
+	"size":                 shared.IsAny,
```

Comment:
Same comment as with storage pools. block.filesystem and size should both have something better than IsAny